AsyncDisplayKit/ASTableView.mm

```diff
@@ -1462,13 +1452,9 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertNodes:(NSA
     }
     [self _scheduleCheckForBatchFetchingForNumberOfChanges:indexPaths.count];
   });
-
-  if (_automaticallyAdjustsContentOffset) {
-    [self adjustContentOffsetWithNodes:nodes atIndexPaths:indexPaths inserting:YES];
-  }
```

Comment:
I guess these lines are deleted based on the assumption that edits are always done in batches and so offset adjustment can be solely done in `-rangeController: didEndUpdatesAnimated:completion:`?

I honestly think we should embrace this "always in batches" strategy to the point that there should be only one delegate method `-rangeController:didUpdateWithChangeSet:`. It will allow us to greatly simplify ASCollectionView and ASTableView. Currently those classes need many internal states to deal with the possibility that updates can be done separately.